### ISSUE TITLE
Model performance

### ISSUE THREAD

**Issue by edwardnguyen1705** [jahongir7174/YOLOv5-tf#1]
Dear @jahongir7174 ,

Thanks for sharing your work.

I am curious about how your train models performance compared to the pytorch version?

**Comment by jahongir7174** [comment]
Thanks for your interest.
I did not train the model with the public dataset, but I trained both Tf and PyTorch models with my custom dataset. PyTorch and TF models have similar results.
I think the model structure is the same in both networks, however, the training strategy is different.

**Comment by edwardnguyen1705** [comment]
Dear @jahongir7174 ,

Thanks for your information. 

Would you mind sharing your checkpoint so that I can check its (`tensorflow` to [OpenVINO](https://github.com/openvinotoolkit/openvino)) performance when converting to [OpenVINO](https://github.com/openvinotoolkit/openvino):, because I face `mAP` drop when running converted (from to `pytorch` to [ONNX](https://github.com/onnx/onnx) to [OpenVINO](https://github.com/openvinotoolkit/openvino)) `yolov5` model on `HDDL` devices, you can take a look at this [issue](https://github.com/openvinotoolkit/openvino/issues/2637).

I also would like to know what is the problem you are working on so I can find a dataset to evaluate the converted model.
Thanks for your time.

**Comment by jahongir7174** [comment]
Sorry for the late answer.
I trained the model with a private dataset. I am planning to train the model with the coco dataset and I will publish the weights.
Thank you

**Comment by Arunavameister** [comment]
@jahongir7174 
Hello, do you have any updates?

**Comment by jahongir7174** [comment]
I did not train the model on public datasets yet.

**Comment by Arunavameister** [comment]
Did you have any success training on your private dataset? If yes, how much time did it take you to train?
I am trying to train on a private dataset myself and the loss just keeps oscillating and even after 200 odd epochs, I cant seem to get the model to train. 
Do you have any ideas? Thanks

**Comment by jahongir7174** [comment]
I might suggest original [YOLOv5](https://github.com/ultralytics/yolov5) for getting a better result
